modules/custom/az_flexible_block/az_flexible_block.info.yml

```
@@ -1,4 +1,5 @@
 name: 'Quickstart Flexible Block'
+description: "A content block with a title and seven dynamic block elements"
```

```suggestion
description: "A content block with a title and dynamic block elements"
```
Less specificity here will make this easier to maintain long term.